cmds/webboot/webboot_test.go

```diff
@@ -205,3 +205,27 @@ func stringToKeypress(str string) []string {
 	}
 	return keyPresses
 }
+
+func TestMirrors(t *testing.T) {
+	uiEvents := make(chan ui.Event)
+	keyPresses := []string{"0", "<Enter>", "0", "<Enter>"}
+	go pressKey(uiEvents, keyPresses)
+	entry := &Config{label: "Arch"}
+	urlResult := ""
+	mirrorNameForTestPurposes := ""
+	var err error
+	urlResult, mirrorNameForTestPurposes, err = mirrorMenu(entry, uiEvents, "")
+	if err != nil {
+		t.Fatalf("Error")
+	}
+	if urlResult != "http://mirrors.acm.wpi.edu/archlinux/iso/2021.06.01/archlinux-2021.06.01-x86_64.iso" {
+		t.Fatalf("Wrong mirror link. The link must be the default one. Expected \"http://mirrors.acm.wpi.edu/archlinux/iso/2021.06.01/archlinux-2021.06.01-x86_64.iso\" but got \"" + urlResult + "\"")
```

Comment:
Put this URL `"http://mirrors.acm.wpi.edu/archlinux/iso/2021.06.01/archlinux-2021.06.01-x86_64.iso"` in a const so it does not need to be typed out twice.